app/templates/events/event_list.html

```diff
@@ -130,56 +138,10 @@ <h2 class="accordion-header" id="accordion__header_{{program}}">
-                data-bs-parent="#categoryAccordion">
-            <div class="accordion-body table-responsive">
-              <table class="table table-striped">
-                <thead>
-                  <tr>
-                    <th scope="col">Event Name</th>
-                    <th scope="col">Date</th>
-                    <th scope="col">Time</th>
-                    <th scope="col">Location</th>
-                    <th scope="col"></th>
-                  </tr>
-                </thead>
-                <tbody>
-                    {% for event in events %}
-                      <tr>
-                          {% if user.isCeltsAdmin or user.isProgramManagerFor(event.program) %}
-                            <td> <a href="/event/{{event.id}}/edit" class="link-primary">{{event.name}}</a></td>
-                          {% else %}
-                            <td> <a href="/event/{{event.id}}/view" class="link-primary">{{event.name}}</a></td>
-                          {% endif %}
-                        <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
-                        <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
-                        <td>{{event.location}}</td>
-                        {% if user.isAdmin %}
-                          <td>
-                            {% if event.isPast %}
-                              {% set isPastEvent = "true" %}
-                            {% endif %}
-                            <button class="btn btn-primary"
-                            onclick="showEmailModal({{event.id}}, {{program.id}}, {{selectedTerm}}, {{isPastEvent}})">
-                              <span class="bi bi-envelope-fill"> Invite</span>
-                            </button>
-                          </td>
-                        {% else %}
-                         <td>
-                          {% if event.id in rsvpedEventsID %}
-                            <button type="button" class="btn btn-danger" onclick="removeRsvpForEvent({{event.id}})">Remove RSVP</button>
-                          {% elif (not event.isPast) and (event.isRsvpRequired) %}
-                            {% set buttonName = "Join Waitlist" if (event.rsvpLimit) and (currentEventRsvpAmount[event.id] >= event.rsvpLimit) else "RSVP" %}
-                            <button type="button" class="btn btn-primary" onclick="rsvpForEvent({{event.id}})">{{buttonName}}</button>
-                          {% endif %}
-                        {% endif %}
-                         </td>
-                      </tr>
-                    {% endfor %}
-                </tbody>
-              </table>
-            </div>
+              class="accordion-collapse collapse {{'show' if programID == program.id else ''}}"
+              aria-labelledby="accordion__header_{{program}}"
+              data-bs-parent="#categoryAccordion">
+              {{createTable(events, "student")}}
```

Comment:
Why is all of this accordion stuff on separate lines?

Reply:
One mega line for you Anderson!